spec/spec_helper.rb

```diff
@@ -29,7 +29,6 @@
 
 RSpec.configure do |config|
   config.before(:suite) do
-    Webservices::Application.model_classes.each(&:recreate_index)
```

Comment:
That's the change I wanted to make.

The rest are fixes for tariff rates specs which were testing queries against the wrong index.